code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -615,6 +615,26 @@
 		/obj/item/crowbar/red/caravan = 1
 	)
 
+/// Mail loot spawner. Some sort of random and rare building tool. No alien tech here.
+/obj/effect/spawner/lootdrop/space/fancytool/engineonly
+	loot = list(
+		/obj/item/wrench/caravan = 1,
+		/obj/item/wirecutters/caravan = 1,
+		/obj/item/screwdriver/caravan = 1,
+		/obj/item/crowbar/red/caravan = 1
+	)
+
+/// Mail loot spawner. Some sort of random and rare surgical tool. Alien tech found here.
+/obj/effect/spawner/lootdrop/space/fancytool/medicalonly
```

Comment:
This makes me 🤨.

This is something that should be extreeeeeeeemely rare, considering you can also use this to get alien surgery from R&D. Having just one alien tool found from space ruins is enough to give you all of this and more. How does the system work to where this is balanced?

Reply:
Adjusted and balanced by diluting the pool even further, current odds are 1-100 for getting alien tools while also requiring the player to actually be selected as a recipient of the letter in the first place.